src/odemis/gui/cont/streams.py

```diff
@@ -217,6 +217,10 @@ def __init__(self, stream_bar, stream, tab_data_model, show_panel=True, view=Non
         if hasattr(stream, "repetition"):
             self._add_repetition_ctrl()
 
+        if self.stream.emitter and hasattr(self.stream.emitter, "applyAutoStigmator"):
```

Comment:
Due to the way Pyro works, `hasattr(self.stream.emitter, "applyAutoStigmator")` is always True! You'd have to call it to find out it actually doesn't exist :-/ 
I can't think of an easy way. On the Delphi, I relied on the fact that .contrast and .brightness existed. Here, that's not even the case. So maybe the best is to look at main.role and check if it's "mbsem".